rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -26,13 +26,14 @@ THE SOFTWARE.
 #include <iostream>
 #include <pybind11/embed.h>
 #include <pybind11/eval.h>
-#include "rocal_api_types.h"
+#include "api/rocal_api_types.h"
 #include "rocal_api.h"
-#include "rocal_api_parameters.h"
-#include "rocal_api_data_loaders.h"
-#include "rocal_api_augmentation.h"
-#include "rocal_api_data_transfer.h"
-#include "rocal_api_info.h"
+#include "tensor.h"
+#include "api/rocal_api_parameters.h"
+#include "api/rocal_api_data_loaders.h"
+#include "api/rocal_api_augmentation.h"
+#include "api/rocal_api_data_transfer.h"
+#include "api/rocal_api_info.h"
```

Comment:
Do we need to do it this way?

In TOT we have not done like that?

Reply:
In TOT we didnt have the files under api/ I think

Reply:
@swetha097 Is this addressed

Reply:
@swetha097 Why this is not addressed yet?

Reply:
reverted this back